go.work

```diff
@@ -0,0 +1,10 @@
+go 1.24.0
+
+toolchain go1.24.4
```

Comment:
Do you need to pin the toolchain to that version for some security measures or anything?

Reply:
ack then!